modules/tinymce/src/plugins/autosave/main/ts/core/Storage.ts

```diff
@@ -71,10 +71,8 @@ const restoreDraft = (editor: Editor) => {
 
 const startStoreDraft = (editor: Editor) => {
   const interval = Settings.getAutoSaveInterval(editor);
-  Delay.setInterval(() => {
-    if (!editor.removed) {
-      storeDraft(editor);
-    }
+  Delay.setEditorInterval(editor, () => {
```

Comment:
This was a fun one... because the interval was never cancelled, it would always keep a reference to the editor.